src/API/Google/AdsAsset.php

```diff
@@ -175,4 +286,29 @@ public function convert_asset( GoogleAdsRow $row ): array {
 			'content' => $this->get_asset_content( $row ),
 		];
 	}
+
+	/**
+	 * Send a batch of operations to mutate assets.
+	 *
+	 * @param MutateOperation[] $operations
+	 *
+	 * @return array A list of Asset's ARN created.
+	 * @throws ApiException If any of the operations fail.
```

Comment:
Maybe we can import `Google\ApiCore\ApiException` allowing to navigate over ApiException in the IDE.